docs/README.md

```diff
@@ -0,0 +1,352 @@
+# Terratest website
+
+This is the code for the [Terratest website](https://terratest.gruntwork.io).
+
+Terratest website is built with Jekyll and published on Github Pages from `docs` folder on `master` branch.
+
+# Quick Start
+
+## Download project
+
+Clone or fork Terratest [repository](https://github.com/gruntwork-io/terratest).
+
+## Run
+
+1. Install [Ruby](https://www.ruby-lang.org/en/documentation/installation/). Version 2.4 or above is recommended.
+   Consider using [rbenv](https://github.com/rbenv/rbenv) to manage Ruby versions.
+
+2. Install `bundler`:
+
+    ```bash
+    gem install bundler
+    ```
+
+3. Go to the `docs` folder:
+
+    ```bash
+    cd docs
+    ```
+
+4. Install gems:
+
+    ```bash
+    bundle install
+    ```
+
+5. Run the docs site locally:
+
+    ```bash
+    bundle exec jekyll serve
+    ```
+
+# Deployment
+
+GitHub Pages automatically rebuilds the website from the `/docs` folder whenever you commit and push changes to the `master`
+branch.
+
+# Working with the documentation
+
+We recommend updating the documentation *before* updating any code (see [Readme Driven
+Development](http://tom.preston-werner.com/2010/08/23/readme-driven-development.html)). This ensures the documentation
+stays up to date and allows you to think through the problem at a high level before you get lost in the weeds of
+coding.
+
+The Terratest website contains *Docs* collection stored in `/docs/_docs`.
+
+When you work with the documentation, it's good to preview the changes. To do that, run project as it is described in [Run section](#run).
+
+1. [A) Change content on the existing page](#change-content-on-the-existing-page)
+2. [B) Add a new page](#add-a-new-page)
+3. [C) Remove or rename page](#remove-or-rename-page)
+4. [D) Add custom redirection](#add-custom-redirection)
+
+## A) Change content on the existing page
+
+1. Find page in `_docs` by file name (it's the same as page's title).
+2. Edit content and save file.
+
+## B) Add a new page
+
+1. Create a new file in `_docs`. The file's name and title have to be the same.
+2. At the beginning of the file, add:
+
+```
+---
+layout: collection-browser-doc                  # X Cannot be changed
+title: Quick start                              # <--- Change this
+categories:
+  - getting-started                             # <--- Change this if needed
+excerpt: Learn how to work with Terratest.      # <--- Change page description
+tags: ["Quick Start", "DRY", "backend", "CLI"]  # <--- Set tags
+order: 100                                      # <--- It sorts the docs on the list
+nav_title: Documentation
+nav_title_link: /docs/
```

Comment:
NIT: I assume these `nav_title_xxx` settings should also not be changed? May want to add comment saying this on the right.